compiler/src/dotty/tools/dotc/core/TypeErrors.scala

```diff
@@ -18,7 +18,7 @@ import config.Printers.cyclicErrors
 class TypeError(msg: String) extends Exception(msg) {
   def this() = this("")
   def toMessage(implicit ctx: Context): Message = super.getMessage
-  override def getMessage: String = throw new Exception("Use toMessage instead for TypeError")
+  override def getMessage: String = super.getMessage
```

Comment:
The call `super.getMessage` will get empty string, which may hide usual error messages.